changelog.json

```diff
@@ -1,5 +1,22 @@
 {
   "releases": {
+    "3.0.6": [
+      "[Added] Add Warp terminal integration for macOS - #14329. Thanks @lhvy!",
+      "[Added] Add context menu to the Current Branch and Current Repository toolbar - #13148. Thanks @uttiya10!",
+      "[Fixed] Older versions of Sublime Text and SlickEdit are also recognized as external editors - #15117. Thanks @vbwx!",
+      "[Fixed] Fix commit shortcut (Ctrl/Cmd + Enter) - #14689. Thanks @tsvetilian-ty!",
+      "[Fixed] Add PyCharm Community Edition support on macOS - #15016. Thanks @tsvetilian-ty!",
+      "[Fixed] Unified diff line gutter context menu items for discard changes no longer enabled when whitespace is hidden - #15003",
+      "[Fixed] 'Show Whitespace Changes' popover appears as expected on unified diff - #15003",
+      "[Fixed] On pull or fetch, make sure the default branch is updated to match the repository settings - #14266",
+      "[Fixed] Show 'Email' label on the preferences form when user is not signed in - #13736. Thanks @andymckay!",
+      "[Fixed] Fix invalid URL state while the \"Clone Repository\" modal is open - #7961. Thanks @tsvetilian-ty!",
+      "[Fixed] Fix commit description with three lines overflowing when it shouldn't - #14791. Thanks @HeCorr!",
+      "[Fixed] Fix notifications on Windows 10 builds prior to the Creators Update - #14714",
+      "[Fixed] 'Update from default branch` menu item allows quick merge of upstream - #14145. Thanks @uttiya10!",
+      "[Improved] On Apple silicon devices running unoptimized builds, auto-update on first run to an optimized build - #14998",
```

Comment:
I just noticed (after deploying the build) that this is only enabled for beta in a feature flag that wasn't at the bottom of the `feature-flags.ts` file, hence why we didn't notice it 😢 

![image](https://user-images.githubusercontent.com/1083228/186375198-140ab0d3-992d-4d66-b74c-fde562f0db6f.png)

I'll remove it from this changelog before merging it and we'll include it in the next release.
```suggestion
```